example/standalone/main.tf

```diff
@@ -4,14 +4,13 @@ module "nifi" {
   # nomad
   nomad_datacenters = ["dc1"]
   nomad_namespace   = "default"
-  nomad_host_volume = "persistence"
 
   # nifi
   service_name    = "nifi"
   host            = "127.0.0.1"
   port            = 8182
+  mode            = "standalone"
```

Comment:
`mode = standalone` must be deleted, since the module `terraform-nomad-nifi` doesn’t have that [input-variable](https://github.com/hannemariavister/terraform-nomad-nifi/blob/master/variables.tf).